src/index.css

```diff
@@ -5,6 +5,17 @@ header {
     width: 100%;
 }
 
+.iframe-container {
+   width: 100vw;
+   height: 100vh;
```

Comment:
Shouldn't this be something like `calc(100vh - headerSize)`?

Reply:
I'm not sure it matters. The content of the iframe is of fixed size, so either the viewport is smaller and we have more overflow, or the user scrolls the page up. I tested it with a 1920x1080 image and it seemed fine. I think we should leave as is, and maybe raise another ticket based on feedback.